src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -1295,19 +1296,20 @@ public UserDetails loadUserByUsername(String username) throws UsernameNotFoundEx
         }
 
         @SuppressFBWarnings(value = "RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE", justification = "Only on newer core versions") //TODO remove when core is bumped
```

Comment:
Could probably clean some of this stuff up in `master` at this point, in a separate PR.

Reply:
⇒ #52